.devcontainer/local-features/bootstrap-winter/codespaces.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+$root = dirname(dirname(dirname(__DIR__)));
```

Comment:
```suggestion
$root = dirname(__DIR__, 3);
```

Reply:
Didn't even know that was a thing - nice!